libs/uw-imap/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (C) 2017 Lucian Cristian <lucian.cristian@gmail.com>
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uw-imap
+PKG_VERSION:=2007f
+PKG_RELEASE:=1
+
+PKG_SOURCE:=imap-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:= \
+	http://ftp.ntua.gr/pub/net/mail/imap \
+	http://gd.tuwien.ac.at/infosys/mail/imap
+
+PKG_HASH:=53e15a2b5c1bc80161d42e9f69792a3fa18332b7b771910131004eb520004a28
+PKG_BUILD_DIR:=$(BUILD_DIR)/imap-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Lucian Cristian <lucian.cristian@gmail.com>
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.txt
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uw-imap
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=University of Washington IMAP toolkit
+  URL:=https://www.washington.edu/imap/
+  DEPENDS:=+libpthread +libopenssl
+endef
+
+define Package/uw-imap/description
+    The UW IMAP tookit includes the following:
+    c-lient library : an API (application programming interface) used to build email clients and servers,
+    including support for IMAP,POP3, SMTP and NNTP protocols and for local mailbox file access on Unix and Windows
+endef
+
+TARGET_CFLAGS += \
+	-DDISABLE_POP_PROXY -fno-strict-aliasing -Wno-pointer-sign -Wno-format \
+	-Wno-implicit-function-declaration -Wno-incompatible-pointer-types
+
+MAKE_FLAGS += \
+	SSLINCLUDE=$(STAGING_DIR)/usr/include/openssl/ SSLCERTS=/etc/ssl/certs \
+	SHLIBBASE=c-client SHLIBNAME=libc-client.so.2007
```

Comment:
Since there are multiple 2007 releases (e.g. 2007f), I'd take the letter's ordinality minus 1, i.e. for 'f' that would be `5 - 1 = 4`, i.e. `2007.4`.

Reply:
ok

Reply:
This is too complicated. I checked what Ubuntu does, and I strongly recommend to do it the same way instead of inventing the wheel again:
```
mhei@kerker:/usr/lib$ ls -la *c-client*
 rw-r--r-- 1 root root 1591548 Okt 26  2014 libc-client.a
lrwxrwxrwx 1 root root      22 Okt 26  2014 libc-client.so -> libc-client.so.2007e.0
lrwxrwxrwx 1 root root      22 Okt 26  2014 libc-client.so.2007e -> libc-client.so.2007e.0
-rw-r--r-- 1 root root 1100216 Okt 26  2014 libc-client.so.2007e.0
```

Reply:
Or we can include the letter, that's fine too.  My point was that `2007` on it's own doesn't uniquely identify a version.

Reply:
ok, so letters or numbers ? I'm in favor of letters, is easier to be spotted

Reply:
2007f works for me.  Just use the value `$(PKG_VERSION)`.